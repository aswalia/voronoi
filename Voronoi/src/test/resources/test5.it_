lft-point: (0,2)
node
lft-point: (5,0)
node
node-point: (5,4) (6,1)
node-point: (6,3) (6.5,2)
lft-point: (7.5,2)
node-point: (8,0) (8,4)
