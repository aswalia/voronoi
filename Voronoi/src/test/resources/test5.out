Vor:
(1.5,5.5) (2.9,2.0)
lft and rgt: (0.0,2.0) (5.0,4.0)
(2.9,2.0) (1.5,-1.5)
lft and rgt: (0.0,2.0) (5.0,0.0)
(2.9,2.0) (4.0,2.0)
lft and rgt: (5.0,0.0) (5.0,4.0)
(4.0,2.0) (6.5,-0.5)
lft and rgt: (5.0,0.0) (6.0,1.0)
(6.5,-0.5) (6.5,-1.5)
lft and rgt: (5.0,0.0) (8.0,0.0)
(7.178571428571429,0.8571428571428572) (6.5,-0.5)
lft and rgt: (6.0,1.0) (8.0,0.0)
(11.75,2.0) (7.178571428571429,0.8571428571428572)
lft and rgt: (7.5,2.0) (8.0,0.0)
(7.0,1.125) (7.178571428571429,0.8571428571428572)
lft and rgt: (6.0,1.0) (7.5,2.0)
(7.0,2.875) (7.0,1.125)
lft and rgt: (6.5,2.0) (7.5,2.0)
(5.25,2.0) (7.0,1.125)
lft and rgt: (6.0,1.0) (6.5,2.0)
(7.0,2.875) (5.25,2.0)
lft and rgt: (6.0,3.0) (6.5,2.0)
(4.0,2.0) (5.25,2.0)
lft and rgt: (6.0,1.0) (6.0,3.0)
(6.5,4.5) (4.0,2.0)
lft and rgt: (5.0,4.0) (6.0,3.0)
(4.0,2.0) (4.0,2.0)
lft and rgt: (5.0,4.0) (6.0,1.0)
(6.5,4.5) (7.178571428571429,3.142857142857143)
lft and rgt: (6.0,3.0) (8.0,4.0)
(7.178571428571429,3.142857142857143) (7.0,2.875)
lft and rgt: (6.0,3.0) (7.5,2.0)
(7.178571428571429,3.142857142857143) (11.75,2.0)
lft and rgt: (7.5,2.0) (8.0,4.0)
(6.5,5.5) (6.5,4.5)
lft and rgt: (5.0,4.0) (8.0,4.0)
(11.75,2.0) (10.0,2.0)
lft and rgt: (8.0,0.0) (8.0,4.0)
